ang/civicase/CaseListTable.html

```diff
@@ -7,16 +8,16 @@
         <thead class="civicase__case-list-table__header">
         <tr>
           <th class="civicase__case-list-column--fixed civicase__case-list-column--first">
-            <span class="case-list-checkbox" ng-if="!viewingCase && selectedCases && bulkAllowed && !item.lock">
-              <input type="checkbox" ng-click="selectAll($event)" id="select-role-{{ $index }}" ng-checked="isSelection('all')" />
-              <label for="select-role-{{ $index }}"></label>
-            </span>
-            <span class="btn-group btn-group-sm" ng-if="!viewingCase && selectedCases && bulkAllowed && !item.lock">
-              <button type="button" class="btn btn-default dropdown-toggle" ng-class="{disabled: !selectedCases.length}" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                {{ ts('Actions') }}
-                <span class="caret"></span>
-              </button>
-              <ul class="dropdown-menu" civicase-actions="selectedCases" multiple="true" refresh-callback="refresh"></ul>
+            <!--  -->
```

Comment:
missing doc

Reply:
Its not missing doc but an empty inline comment missed . Let me remove it.